src/domain/collection/curatedEvents/utils.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { toast } from 'react-toastify';
+
+import {
+  EventFieldsFragment,
+  LinkedEventsSource,
+  useEventsByIdsQuery,
+} from '../../../generated/graphql';
+import { isEventClosed } from '../../event/EventUtils';
+import { EventType } from '../../event/types';
+
+export const PAGE_SIZE = 10;
```

Comment:
that was the copy-paste thing? and `utils.tsx` name could be more descrptive perhaps?

Reply:
renamed

Reply:
PAGE_SIZE still sits there

Reply:
I actually don't know what do you want me to do to it, didn't I already delete the duplicate?

Reply:
`eventSearch/constant.ts` still have the same value? 
![Screenshot 2021-03-31 at 10 35 51 AM](https://user-images.githubusercontent.com/7648194/113107973-2b15a100-920d-11eb-85fa-c8d9263963ff.png)

Reply:
Well that is totally different PAGE_SIZE. One is for event colleciton pagination and another one is for event search list page size